sdk/nodejs/tests/runtime/testdata/closure-tests/cases/152-Capture-config-created-on-the-inside/snapshot.txt

```diff
@@ -1,13 +1,17 @@
 exports.handler = __f0;
 
+var __deploymentOnlyModule = {};
```

Comment:
Changes here are due to `export *` compiling slightly different in 3.9 and later.